### PR TITLE
Collect container logs for k8s control plane and haproxy

### PR DESCRIPTION
**Pull request by diamonwiggins**

### CODE REVIEW

File: host/cluster-down.yaml [hunk 1]
```diff
@@ -185,6 +185,48 @@ spec:
         collectorName: "journalctl-dmesg"
         command: "journalctl"
         args: ["--dmesg", "--no-pager", "-S", "7 days ago"]
+    # Logs for K8s Control Plane
+    - run:
+        collectorName: "crictl-logs-apiserver"
+        command: "sh"
+        args: ["-c", "crictl logs $(crictl ps -a --name apiserver -l | awk 'NR!=1 {print $1}') 2>&1"]
```
Comment (adamancini) [inline host/cluster-down.yaml:192]: `NR!=1` is that to skip the first line where the headers are printed?

Comment (adamancini) [reply host/cluster-down.yaml:192]: otherwise looking pretty good to me

**Review by adamancini — APPROVED** [review]

File: host/cluster-down.yaml (hunk 1 above)
Comment (diamonwiggins) [reply host/cluster-down.yaml:192]: > NR!=1 is that to skip the first line where the headers are printed?

It is

Comment (banjoh) [reply host/cluster-down.yaml:192]: How about replacing `| awk 'NR!=1 {print $1}'` with `--quite` that way this works in those rare cases where `awk` does not exist?

The command would look like `crictl logs $(crictl ps -a --name apiserver -l --quiet`

**Pull request merged**